packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/utils/normalizeRelations.js

```diff
@@ -43,6 +36,13 @@ export const normalizeRelations = (
         ]
           ?.map((page) =>
             page?.results
+              .filter((relation) =>
+                modifiedData?.disconnect?.find(
+                  (disconnectRelation) => disconnectRelation.id === relation.id
+                )
+                  ? null
+                  : relation
```

Comment:
I think since you are returning `modifiedData?.disconnect?.find()` here, you don't have to return `null` or `relation` and can just use the result of `.find()` because `.filter()` operates on `booleans`.

I'd suggest

```js
.filter((relation) =>
  !modifiedData?.disconnect?.find(
    (disconnectRelation) => disconnectRelation.id === relation.id
  )
)

```

Reply:
brain was so broken yesterday 🤦‍♀️